vue3/vite.config.js

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from 'vite';
+import vue from '@vitejs/plugin-vue';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [
+    vue({
+      template: {
+        compilerOptions: {
+          // isCustomElement: (tag) => tag.includes('gcds'),
```

Comment:
Can this comment be removed?

Reply:
I changed it and added a comment instead for now, LMK what you think!